app/views/teacher_interface/english_language/edit_exemption.html.erb

```diff
@@ -0,0 +1,52 @@
+<% content_for :page_title, "#{"Error: " if @form.errors.any?}#{t(".#{@exemption_field}.heading")}" %>
+<% content_for :back_link_url, back_history_path(default: teacher_interface_application_form_path) %>
+
+<span class="govuk-caption-l"><%= t("application_form.tasks.sections.english_language") %></span>
+<h1 class="govuk-heading-l"><%= t(".#{@exemption_field}.heading") %></h1>
+
+<p class="govuk-body"><%= t(".#{@exemption_field}.description") %></p>
+<p class="govuk-body"><%= t(".#{@exemption_field}.evidence") %></p>
+
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-one-half">
+    <ul class="govuk-list govuk-list--bullet">
+      <li>Antigua and Barbuda</li>
```

Comment:
Should we extract these somewhere as we've listed them a few times now?

Reply:
Yeah, good idea, will do that!